testsuite/domain/pom.xml

```diff
@@ -305,6 +305,41 @@
                 </plugins>
             </build>
         </profile>
+
+        <profile>
+            <id>ci-cleanup</id>
+            <activation>
+                <property>
+                    <name>cleanup</name>
+                </property>
+            </activation>
+            <build>
+                <plugins>
+                    <!-- Clean the server configuration(s) -->
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-clean-plugin</artifactId>
+                        <executions>
+                            <execution>
+                                <id>clean-manual-mode-servers</id>
```

Comment:
This id is repeated in all the poms, so if it's not going to be unique per pom it shouldn't mention manual-mode.